Documents/Annotations/AnnotationsStyling/MainWindow.xaml

```diff
@@ -77,7 +77,7 @@
                     </ResourceDictionary.MergedDictionaries>
                 </ResourceDictionary>
             </FlowDocumentPageViewer.Resources>
-            <FlowDocument>
+            <FlowDocument AutomationProperties.Name="Sample Text">
```

Comment:
nit: Paragraph is a DependencyObject.  Should the AutomationProperties go there instead of FlowDocument?

Reply:
Paragraph doesn't have an associated AutomationPeer that would allow us to set AutomationProperties on it.